pkg/autoscaler/async_scaler.go

```diff
@@ -126,5 +126,5 @@ func (s *AsyncScaler) CurrentReplicas(apiName string) (int32, error) {
 		return 0, errors.ErrorUnexpected("unable to find k8s deployment", apiName)
```

Comment:
nit: rename `CurrentReplicas` to `CurrentRequestedReplicas`

Reply:
the method is satisfying an interface